src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -1,166 +1,72 @@
-
-    if (enabled) {
-      const fcmToken = await messaging().getToken();
-      const payload = {
-        fcm_token: fcmToken
-      };
-      fcmTokenService(payload);
-    }
-  };
+  const {
+    channels: anonChannels,
+    goToChatScreen,
+    goToPostDetailScreen,
+    goToContactScreen
+  } = useAnonymousChannelListScreenHook();
 
   React.useEffect(() => {
-    const checkNotif = async () => {
-      if (!navigation.isFocused()) return;
-      PushNotification.checkPermissions(
-        (
-          data: PushNotificationPermissions & {
-            authorizationStatus?: number;
-          }
-        ) => {
-          const lastPromptTime = StorageUtils.lastPromptNotification.get();
-          const promptInterval = 48 * 60 * 60 * 1000;
-          const currentTime = new Date().getTime();
-          if (lastPromptTime && currentTime - parseFloat(lastPromptTime) < promptInterval) {
-            return; // Don't show the prompt if the interval has not passed
-          }
-
-          const needsPermission =
-            data.alert === false || data.badge === false || data.sound === false;
+    if (isFocused) {
+      refresh('channelList');
+      checkNotificationPermission();
+    }
+  }, [isFocused]);
 
-          const showAlert = (onPressAction: () => void) =>
-            Alert.alert(
-              "Don't Miss New Messages",
-              `Allow notifications to know 
-when friends send you messages.`,
-              [
-                {text: 'Not now', onPress: () => console.log('Cancel Pressed')},
-                {text: 'Allow', onPress: onPressAction, isPreferred: true}
-              ]
-            );
+  const renderChannelItem = ({item}) => {
+    if (item?.channelType === 'ANON_PM') {
+      return <MessageChannelItem item={item} onChannelPressed={() => goToChatScreen(item)} />;
+    }
 
-          switch (data.authorizationStatus) {
-            case PERMISSION_STATUS_ACCEPTED:
-              if (needsPermission) {
-                // THIS CASE IT'S TRIGGERED WHEN USER MANUALLY DISABLED THE NOTIFICATION
-                StorageUtils.lastPromptNotification.set(currentTime.toString());
-                showAlert(() => openSettings().catch(() => console.warn('cannot open settings')));
-              } else {
-                requestPermission();
-              }
-              break;
-            case PERMISSION_STATUS_BLOCKED:
-              if (needsPermission) {
-                StorageUtils.lastPromptNotification.set(currentTime.toString());
-                showAlert(() => openSettings().catch(() => console.warn('cannot open settings')));
-              }
-              break;
-            case PERMISSION_STATUS_PENDING:
-              StorageUtils.lastPromptNotification.set(currentTime.toString());
-              showAlert(() => requestPermission());
-              break;
-            default:
-              break;
-          }
-        }
+    if (item?.channelType === 'ANON_POST_NOTIFICATION') {
```

Comment:
Should be using constant variable for `ANON_POST_NOTIFICATION`

Reply:
solved in new commit